x-pack/plugins/task_manager/server/task_validator.ts

```diff
@@ -57,13 +57,14 @@ export class TaskValidator {
       return task;
     }
 
+    const taskTypeDef = this.definitions.get(task.taskType);
+
     // In the scenario the task is unused / deprecated and Kibana needs to manipulate the task,
     // we'll do a pass-through for those
-    if (!this.definitions.has(task.taskType)) {
```

Comment:
surprised there's no unit test for validating an unknown task type to update. Maybe we should add one?

Reply:
Do these not cover that?

https://github.com/elastic/kibana/blob/e49e89e5753893d9f3bdff0d23b84e45b384bba2/x-pack/plugins/task_manager/server/task_validator.test.ts#L29-L40

https://github.com/elastic/kibana/blob/e49e89e5753893d9f3bdff0d23b84e45b384bba2/x-pack/plugins/task_manager/server/task_validator.test.ts#L287-L298